src/fib_payments/utils.py

```diff
@@ -0,0 +1,71 @@
+import asyncio
```

Comment:
File Rename: Please rename `utils.py` to `async_http_utils.py` to better reflect the purpose of the file, which contains asynchronous HTTP utilities and retry logic